documentation/public/user-guide/06-query-language/01-query-syntax/index.md

```diff
@@ -9,46 +15,46 @@ optionally, a series of other patterns that match the node's children. For
 example, this pattern would match any `MultiplicativeExpression` node that has
 two children `Expression` nodes, with an `Asterisk` node in between:
```

Comment:
```suggestion
two children [`Expression`](../../../solidity-grammar/05-expressions/01-base-expressions/) nodes, with an `Asterisk` node in between:
```

Reply:
I can't directly map the `Asterisk` node, as in the Grammar section it's named in UPPERCASE. Perhaps this deserves an explanation.